pyaedt/__init__.py

```diff
@@ -1,14 +1,50 @@
 # -*- coding: utf-8 -*-
 import os
+import sys
+import warnings
 
 if os.name == "nt":
     os.environ["PYTHONMALLOC"] = "malloc"
 
-pyaedt_path = os.path.dirname(__file__)
+LATEST_DEPRECATED_PYTHON_VERSION = (3, 7)
 
-__version__ = "0.8.dev0"
 
+def deprecation_warning():
```

Comment:
Maybe we could rename it `deprecation_python_warning` to avoid confusion with other deprecation warnings methods in the code. But it is true that the current function is located in the `init` file. So confusion is mitigated.